dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ItemRestRepository.java

```diff
@@ -92,6 +93,9 @@ public class ItemRestRepository extends DSpaceObjectRestRepository<Item, ItemRes
     @Autowired
     RelationshipTypeService relationshipTypeService;
 
+    @Autowired
+    IdentifierService identifierService;
```

Comment:
Unless I missed something, this service seems not used anymore

Reply:
correct, removed